conf/modules/call_repeat_expansions.config

```diff
@@ -43,27 +43,12 @@ process {
     }
 
     withName: '.*CALL_REPEAT_EXPANSIONS:BCFTOOLS_REHEADER_EXP' {
-        ext.args = "--temp-prefix ."
-        ext.prefix = { "${meta.id}_reheader" }
-    }
-
-    withName: '.*CALL_REPEAT_EXPANSIONS:RENAMESAMPLE_EXP' {
+        ext.args = "--temp-prefix . -O v"
+        ext.prefix = { "${meta.id}_repeat_expansion" }
         publishDir = [
             path: { "${params.outdir}/repeat_expansions" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null :
-                filename.contains('_renam.vcf.gz') ? "${meta.id}_repeat_expansion.vcf.gz" :
-                filename },
-        ]
-    }
-
-    withName: '.*CALL_REPEAT_EXPANSIONS:TABIX_EXP_RENAME' {
-        publishDir = [
-            path: { "${params.outdir}/repeat_expansions" },
-            mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null :
-                filename.contains('_renam.vcf.gz.tbi') ? "${meta.id}_repeat_expansion.vcf.gz.tbi" :
-                filename },
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
     }
```

Comment:
@jemten I am thinking if it might be better to add a parameter to generate reviewer compatible output files, like we are doing for cram files at the moment...  Then we can publish the compressed files by default, and we can choose to publish the uncompressed files based on whether that parameter is set to true or not.

Reply:
Good suggestion!